File: psets/pset6/mario/more/mario.py
from cs50 import get_int

# Prompts the user for the height
while True:
    h = get_int("Height: ")

    if h > 0 and h <= 8:
        break

# Prints the pyramid
for i in range(h):
    # Prints the left spaces
    for s in range(h - (i + 1)):
        print(" ", end="")

    # Prints the left hashes
    for p in range(i + 1):
        print("#", end="")

    # Prints the gap spaces
    print("  ", end="")

    # Prints the right spaces
    for p in range(i + 1):
        print("#", end="")

    # Changes the line
    print()

File: psets/pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;

    do                                         //prompts the user for the height
    {
        h = get_int("Height: ");
    }
    while (h <= 0 || h > 8);

    for (int i = 1; i <= h; i++)               //prints the pyramid
    {
        for (int s = 0; s < h - i; s++)        //prints the left spaces
        {
            printf(" ");
        }

        for (int p = 0; p < i; p++)            //prints the left hashes
        {
            printf("#");
        }

        printf("  ");                          //prints the gap spaces

        for (int p = 0; p < i; p++)            //prints the right hashes
        {
            printf("#");
        }

        printf("\n");                          //changes the line
    }
}